### DISCUSSION TITLE
Add ability to move breakpoint to previous line

### DISCUSSION THREAD

**Discussion by duyn9uyen** [OmniSharp/omnisharp-vscode#6262] [Ideas]
In `Visual Studio`, I can move the current break point back to the previous line. In VS Code, there is a `Run to cursor` command. Are there any plans to allow `Run to previous cursor`? It can take quite some time to restart debugging my app. With the plans to retire `Visual Studio Mac`, you will have a lot of new VS Code users. This feature has always been missing in VS Code compared to Visual Studio.  Can you add an ability to move the breakpoint to previous line?

**Comment by gregg-miskelly** [comment]
Visual Studio Code calls this feature "Jump to Cursor". Right click on the line you want to execute next, and invoke it. If you find it more convenient to be able to drag the instruction pointer, you could open a suggestion issue against VS Code. This isn't something an extension could implement.

![image](https://github.com/dotnet/vscode-csharp/assets/11078489/9c16da7b-623c-4dab-ba3c-c1bb033ee940)

**Reply by duyn9uyen** [reply]
Ah. The `Jump to Cursor` only works if your cursor is in the **beginning** of a line. Not at the end of a current line. Thanks.
